ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -385,7 +427,12 @@ static int HandleVnodeOperation(
         {
             if (FileFlagsBitIsSet(currentVnodeFileFlags, FileFlags_IsEmpty))
             {
-                functionSample.SetProbe(Probe_VnodeOp_HydratePlaceholderFile);
+                if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, &root, &vnodeFsidInode, &kauthResult, kauthError))
```

Comment:
Can\should the calls to `TryGetVirtualizationRoot` be moved into `TrySendRequestAndWaitForResponse`?

Reply:
That makes sense.   Given the choice I'd rather go with whatever is cleanest\easiest with the profiling code (so if keeping this out makes the most sense I'm good with that 😄)